qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/BackupTest.java

```diff
@@ -74,14 +81,79 @@ void setup() {
   @Test
   void shouldBackup() {
     // given
-    client.createSingleJob("test");
+    client.createSingleJob(JOB_TYPE);
 
     final long backupId = 2;
 
     // when
     backup(backupId);
 
     // then
+    assertBackupCompleted(backupId);
+  }
+
+  @Test
+  void shouldRestore() throws InterruptedException {
```

Comment:
Which PR are you talking about. I saw BackupApiIT based on test containers. This branch already contains that. But I was thinking that to add different test scenarios described [here](https://github.com/camunda/zeebe/issues/10387), it would be more flexible to do it without containers. I did not try to do it with TestContainers yet, so may be it is not impossible as I assumed.

Reply:
It's been trying to get merged for a full day already but flaky issues (tests and also runner issues :smile:), you know...

https://github.com/camunda/zeebe/pull/10418

I initially deleted it because what it was so far was mostly just overlapping the `BackupApiIT`, but as I said, if we want to test different things here we can do so as well :+1: I think it'll just be important to highlight the differences and what tests go where, as otherwise it may not be immediately apparent to everyone

Reply:
Ok. I thought we [discussed](https://camunda.slack.com/archives/C037RS2JHB8/p1663664501292299?thread_ts=1663662783.435509&cid=C037RS2JHB8) about this. 

Our PRs are going to conflict. So I can also remove this test now. And add it again as part of #10387

Reply:
I removed this test. I wanted to rewrite it anyway as part of #10387 . So this PR will be merged without any test :wink:

Reply:
Sorry about that, I think I misunderstood and thought we would do that as part of #10387, and in the mean time go with the testcontainers one :see_no_evil: